ckanext/canada/auth.py

```diff
@@ -27,6 +31,14 @@ def datastore_upsert(up_func, context, data_dict):
     return up_func(context, data_dict)
 
 
+@chained_auth_function
+@auth_allow_anonymous_access
+def user_create(up_func, context, data_dict=None):
+    # additional check to ensure user can access the Request an Account page
+    # only possible if accessing from GOC network
+    goc_auth()
```

Comment:
@RabiaSajjad the auth methods cannot contain the Flask request and aborts as they are used outside of the Request context as well.

So this will just need to do something like:
```
if not registry_network_access():
        return {'success': False}
return up_func(context, data_dict)
```